### DISCUSSION TITLE
Atari 7800 module compatibility

### DISCUSSION THREAD

**Discussion by MrMegaManFan** [OnionUI/Onion#1112] [General]
After discussing the issues I've had with the standard Atari 7800 emulation package with the RetroArch implementation on Onion OS for Miyoo Mini (+), specifically games created by Bob (PacManPlus) on AtariAge.com, I was wondering if a better package could be offered in a future update. He has released all of the roms for his homebrews for free so this would be easy to test. Thank you.

**Comment by wfg97079** [comment]
Seems to have issues with lots of 7800 hombrew.  Any chance of updating which emulator is used for 7800?
